lib/src/main/java/uk/gov/di/ipv/cri/drivingpermit/library/config/ParameterStoreParameters.java

```diff
@@ -4,58 +4,56 @@
 
 public class ParameterStoreParameters {
 
-    public static final String DCS_HTTPCLIENT_TLS_CERT = "/dl-cri-api-v1/DCS/HttpClient/tlsCert";
+    public static final String DCS_HTTPCLIENT_TLS_CERT = "DCS/HttpClient/tlsCert";
```

Comment:
Should the existing DCS values not match the current naming? I dont think we want to rename all the DCS vars

Reply:
So my thinking was that once we have DVA AND DVLA it's gonna start to get messy/confusing with the certs, explicitly saying it's DCS will make it easier to tell the difference, also keeps it inLine with what is in Passport

Reply:
That being said I understand the risk with changing the parameters over so have left them in this file for now and kept the values that are being pulled in to the configuration service to be the original ones